### ISSUE TITLE
Meeting Minutes

### ISSUE THREAD

**Issue by ththelen** [NCSU-CHAZ/Sunny-Day-Flooding#11]
Summary of items discussed at meetings. Emphasis on action items.

**Comment by ththelen** [comment]
**August 18 - TT, KA, CD**

_Discussion items_
Follow-up on lingering items from NC9rev run
Water level offset: .21m vs .15m
Flood event to use for comparison with SuDS data: May 13-17

_Action items_
TT: 
- Plot 4-5 synthetic stations through Cape Fear/Snow's Cut and Tidal Inlet/Intracoastal to drill down on tidal propagation
- Set modeling goals to complete this fall prior to AGU
- Start figuring out 3Di model
- Visualize +0.21m run results, and +0.15m run results once CB shares

CD:
- Complete +0.15m run for NC9 and NC9rev meshes
 
All (@anardek and @caseydietrich):
- Decide on meeting time for fall semester. It has been useful getting all three of us together on a bi-weekly(ish) basis. I would like to continue this through the fall semester if you can make it work in your schedules. I am generally available outside of the coastal team and SunnyD project meetings on Wednesdays and your respective classes on Tuesday/Thursday. Please reply to this thread with your availability for fall meetings and any suggestions to change the meetings to make them more effective.

**Comment by ththelen** [comment]
**Semester Goals - Running List**
- Figure out couple between ADCIRC, 3Di
- Finalize Nov 2021 runs, fine tune ADCIRC model
- Model May 2022 flood events in ADCIRC, 3Di coupled model & present these results at AGU. 3Di model may not be fine-tuned yet, but that is okay.

**Comment by ththelen** [comment]
**Meeting Agenda: September 1**

- Discuss tidal propagation time series for Cape Fear/Snow's Cut and Tidal Inlet/Intracoastal
- Discuss +0.15m model results and decide on next steps (if any) for ADCIRC mesh refinement

**Comment by ththelen** [comment]
**Meeting Minutes: September 1 - TT, KA, CD**

_Discussion Items_
ADCIRC / 3Di model coupling boundary, methods (WL vs discharge)
Semester goals (have at minimum refined ADCIRC model to present at AGU)
Discussion of 3 tidal propagation routes
Review of time series and RMSE from all recent model runs

_Action Items_
TT:
- Learn to plot model results using Python. Coordinate with Jenero
- Learn how to add Cape Fear River BC to NC9
- Visualize CoNED bathymetry and mesh bathy/resolution at CB tidal inlet

CD:
-  Provide TT with review class material for adding Cape Fear BC to NC9
- (Later) meeting with TT to work through adding Cape Fear Rive BC with new SMS 13.2

**Comment by ththelen** [comment]
**Meeting Minutes: Sept 15 - TT, KA, CD**

_Discussion Items_
Review of bathymetry and mesh representation around tidal inlet.
Discussion of Cape Fear River BC.
Brief recap of 3Di convo.

_Action Items_ 
TT:
- Replot representation of tidal inlet elevation below 0 NAVD88 with color breaks every meter. Use better color scale provided by @caseydietrich 
- Trim down mesh for Cape Fear River BC. Identify long, straight, constant depth channel section near river gauge to introduce flow to mesh.
- Remesh the larger location around the Tidal Inlet. 
- Continue progressing plotting notebook.

CD:
- Provide tutorial for remeshing around the Tidal Inlet.
- Provide preferred SMS color scale.

**Comment by ththelen** [comment]
**Meeting Minutes: Sept 29 - TT, KA, CD**
_Discussion Items_
Review of updates in NCrev2.0, NCrev2.1
Review of data plotting progress

_Action Items_
CD:
- Provide 1st leg of NC9rev2.1 simulation to TT to check that Cape Fear River flow BC is working as expected.
- Schedule session with TT to hand over running ADCIRC simulations. Items to discuss include: WL offset, river flux BC, atmospheric forcing.

TT:
- Identify outlier detection algorithm to automatically clean FIMAN data (e.g., clean periodicity < 6hrs). Can ask @anardek for direction.
- Visualize 1st leg of NC9rev2.1 simulation to check that Cape Fear River flow BC is working as expected. See https://github.com/NCSU-CHAZ/Sunny-Day-Flooding/issues/17#issuecomment-1259558893
- Take over ADCIRC simulation running from @caseydietrich after handoff session.
- Plot NC9rev2.0 time series, potentially to share at conferences if it is an improvement over NC9rev.
- Outline NAF presentation for to share with KA, CD on 10/13 modeling meeting. 15 minute oral presentation should respond directly to the prompt Making Climate Science Usable and Useful. Full prompt below:
_Making climate-informed decisions requires access to relevant, credible, and legitimate climate information. Access includes the ability to identify, obtain, understand, and apply climate science in decision-making. This session will focus on the variety of tools, technologies, and approaches to climate data translation and how they are most effectively being used. We are accepting abstracts for presentations with a particular focus on co-produced climate science and its delivery, as well as tools that have been evaluated for their efficacy and use in perpetuity._

**Comment by anardek** [comment]
Thomas, let's try just low pass filtering for now and see if that removes
all the outliers you've identified.

**Comment by ththelen** [comment]
**Meeting Minutes: 10/13 - TT, KA, CD**
_Discussion Items_
NAF Presentation Outline review
NC9rev2.0 and rev2.1 results discussion

_Action Items_
TT:
- Check for tidal signal at Cape Fear River USGS gauge
- Run NC9rev2.2 (improved river bathy or river BC moved downstream to realistic bathy)
- Run NC9rev2.2 with 1.67 flux multiplier (b/c CFR drains 60% of watershed to CFRE)
- Run NC9rev3.0 with higher node density in tidal inlet, intracoastal
- Edit NAF presentation
- Prep 3Di friction raster

KA @anardek:
- Provide budget from CB for stormwater / bulkhead improvements on Florida Ave.

**Comment by ththelen** [comment]
**Meeting Minutes: 11/10 - TT, KA, CD**
_Discussion Items_
Changes in NC9rev2.2: raise elevations along TI north bank, keep an eye on stability of skinny element along IC bank
May 2022 event: could shorten spinup period
AGU slides: want to show May 2022 event

_Action Items_
TT: keep progressing NC9rev2.2, May 2022 event

**Comment by ththelen** [comment]
**Meeting Agenda: 12/7 - TT, KA, CD**
Hello @anardek and @caseydietrich , I am circling back with an agenda for our 5 pm EST meeting this Wednesday. We have a lot to cover in this meeting, so I want to make one more plug for skimming the Nov 2021 and May 2022 summary of results documents prior to the meeting. I think we will have a better chance to cover all the discussion items it you have already perused the water level, wind, and wave model validation summaries. I am particularly interested in any big-picture model skill takeaways that you are able to synthesize from the Nov and May event data.

_Discussion Items_
Nov 2021 model validation (WL, wind, waves) summary and review of NC9rev2.2 performance (summary doc: https://github.com/NCSU-CHAZ/Sunny-Day-Flooding/issues/21#issuecomment-1325579564)
May 2022 model validation summary (summary doc: https://github.com/NCSU-CHAZ/Sunny-Day-Flooding/issues/20#issuecomment-1325576175)
3Di training and model development updates
Results to share at AGU

**Comment by ththelen** [comment]
**Meeting Summary: 12/7 - TT, CD**
Summary of meeting takeaways for @anardek (cc: @caseydietrich)

_Discussion Items_

**Nov 2021 and May 2022 flood events**
Wind and wave data are reassuringly accurate to gauge data. We underpredict wave height for the May event, but this may be because SWAN more accurately simulates larger wave heights than smaller ones. Tidal propagation errors from poor channel resolution do not seem to be the cause of small water level inaccuracies because we see the same inaccuracies at open ocean (Wrightsville) and inland (Carolina Beach & Wilmington) gauges. We did not see any improvement in water level accuracy to gauge data with the remeshed NC9rev2.2, so I will proceed with modeling on the original resolution rev2.1 mesh. 

Overall, the results indicate a level of accuracy for both tide and wind-driven events that should be defensible in an academic journal, so TT & CD suggest that we proceed with NC9rev2.1 (original resolution, select hand-edits to bathymetry/topography, cut down floodplains for faster simulation times in future runs) for use in coupled model simulations. It seems like we have reached the point of diminishing returns on mesh changes and/or adding additional forcings in ADCIRC. Also, TT should find a more condensed way to share Jupyter notebook results that hides code -- apologies for the long-form documentation.

**3Di model updates**
Including the inlet in the 3Di model makes sense to accurately represent CB waterway hydrodynamics. Next step in model development is building out the CB stormwater system.

*Results to share at AGU**
CD: ADCIRC section looks good - dual driver May results are interesting.

**Comment by ththelen** [comment]
Good morning @anardek and @caseydietrich, I would like to request a meeting to discuss the events I have proposed to model for the paper and settle on a framework for completing the model runs. What I will prepare for this meeting is the powerpoint with proposed events (linked here: https://github.com/NCSU-CHAZ/Sunny-Day-Flooding/issues/10#issuecomment-1404279019) and a draft plan of action to complete the simulations. This plan will include details on run duration (e.g., days simulated, spin up period) and items that I need from you to complete the simulations (e.g., NAM file prep). 

I suggest sometime within Katherine's 2-5 pm meeting block this Thursday, perhaps the 2-3 pm time slot we met during a couple weeks ago. Please let me know what you think about this proposal.

**Comment by caseydietrich** [comment]
@ththelen Sure, I can do Thu 2pm for this meeting. Thanks for pushing on this. (cc: @anardek )

**Comment by caseydietrich** [comment]
@ththelen I reserved FWH 3103 at that time.

**Comment by anardek** [comment]
I can also do 2 pm on Thursday.

**Comment by ththelen** [comment]
**Meeting Minutes: 3/2/2023 - TT, CD, KA**

_Discussion Items: results from ADCIRC (tides only) and SWAN+ADCIRC (tides + atmospheric) runs for 3 events_

June 2022 event - primary driver: tides
- Not concerned about slight underprediction of water levels on June 15 since water levels on all other days are accurate. Hypothesis: NAM forcing did not accurately represent the atmospheric conditions that day .

August 2022 event - primary driver: rain
- SWAN + ADCIRC water levels lag measured data by ~ 1 hour, but otherwise simulation results look good.

January 2023 event - mixed drivers
- NOAA observed - predicted water levels not a good proxy for finding the ADCIRC water level offset. Hypothesis: there is too much NTR noise increasing the observed water level signal for this averaging technique to produce an offset that accurately depicts WL contributions that we are missing in ADCIRC (e.g., longer term processes like the Gulf Stream)
- Alternate strategy from CD: Run ADCIRC tides + atmospheric simulation for the prior month, then calculate the offset as follows: WL offset = from prior month avg(ADCIRC tides + atmospheric) - avg(FIMAN). See how much your simulation was missing by the month before, then adjust January runs by this amount.

General notes:
- Start preparing plots to publication standard. Gives variable names that will be used in paper (CB01 = clamshell, FIMAN = harbor, ADCIRC = tides only, think about color scales)

Action Items
TT: 
- Run ADCIRC tides + atmospheric simulations for June, August events
- Run 3Di simulation. Start with August event
- Run simulation to WL offset for January event.

@caseydietrich: 
- Provide fort.22* NAM forcing files for month of December 2022.

**Comment by ththelen** [comment]
**9/5/2023 - Structure Control Check-in Meeting Slide Deck**

https://docs.google.com/presentation/d/1O4wuresfqCe2wLs-vXnIa0Xe97EYOQ9rAsQdax_btDA/edit?usp=sharing

**Comment by ththelen** [comment]
**Meeting Minutes: 9/6/2023 - TT, KA, CD**

_Discussion Items: how to model duckbills_

General notes: 
- Not worth further manipulating weir parameters to try to match seepage rates into catch basins since we already accurately represent flood water depths/timing, the important part of the signal.

_Acition Items_
TT:
- Add infiltration to model
- Decide how to represent sensor, pipe outfall, weir elevations
- Write supplement to describe how 3Di weirs are used to represent duckbills. Describe strengths (C parameter tuning) and weaknesses (change in weir flow regime), but note no models currently have a good method to simulate these structures.
- Complete model run figures, decomps

**Comment by ththelen** [comment]
**Meeting Minutes: 9/25/2023 - TT, KA, CD**

_Discussion Items: time series and decomposition measured and model results_

General notes:
Need section that describes Carolina Beach flooding mechanisms solely using SuDS data and photos preceding model results. This section should include images of the three kinds of floods we model, using photos from the actual events if possible (e.g., August 2022 image for rain-driven flood). This section should also contain the observation the "backflow prevention devices modify the tidal signal in stormwater systems."
- Quantify how many floods were expected vs unexpected (infer contribution from rain or atmospheric) based on the CB flood alert benchmark. Maybe include this in the section above.
- Be clear: WL signal in catch basins includes both contributions from leakage through valve and groundwater.

Notes on figures:
- Remove data pts when drain is dry or water levels are below SuDS elev. Use Adobe or mask data pts within x meters of SuDS elev.
- Decomposition y-axis label "Water level (m above outfall)"
- Be explicit about plot duration in captions "2 hour timeseries showing ..."
- Make plots 5-6 figs vertically stacked. Use shaded background to show zoomed in image for time series detail. 
- Show SuDS, outfall, road elevs in very faint gray. Only color should be in timeseries, decomp lines.
- See if changing spacing of dots, dash-dots makes lines easier to see. Other plot decompositions in solid color lines.
- Switch from "winds" in plot label to "atm." 

_Action Items_
TT:
- Finish all figures for all section of paper.
- Place all figures and captions in slide deck. Share with KA, CD for review.
